### DISCUSSION TITLE
[吐槽] Logo 文字 ts 中的 s 与 cat 的安排是巧合吗

### DISCUSSION THREAD

**Discussion by mslxl** [NapNeko/NapCatQQ#540] [General]
![image](https://github.com/user-attachments/assets/bbe89ead-7938-4e9a-95d1-d317c5b8cd57)

有关 scat 的含义：
![image](https://github.com/user-attachments/assets/fbdfe4f1-3101-4e02-9294-630dd4e9b4a0)

**Comment by MliKiowa** [comment]
也许是的( 但是scat的歧义有这么多.jpg

**Reply by mslxl** [reply]
~~其实在某个知名漫画网站上还挺常见的，过于重口给我留下了不可磨灭的印象~~

虽然从百科和词典上看起来含义很多，但是从Google搜索的结果上可以看到目前常见的意思只有这个含义、`离开`和`一种歌唱方式`。

---

当然我也就小小吐个槽，毕竟NapCat的核心是软件而不是艺术作品
